api/src/Api/Request/Data/TotalSocial/Import/ImportMetricListApiRequestData.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Api\Request\Data\TotalSocial\Import;
+
+use App\Api\Provider\MetricIntervalProvider;
+use App\Api\Request\Data\ApiRequestDataInterface;
+use App\Api\Services\VariableNameConverter;
+
+class ImportMetricListApiRequestData implements ApiRequestDataInterface {
+
+    public $ids = [70905, 70933];
+    public $metrics = 'offline.scoreVolume.value';
+    public $outputType = 'overtime';
+    public $from = 1442188800;
+    public $to = 1543190399;
```

Comment:
November 25th, 2018 23:59:59 for this value and like the $from, you seems to not used it

Reply:
the same as above, I forgot to remove it